external-api/src/types.rs

```diff
@@ -105,6 +111,7 @@ impl From<IndexedWallet> for Wallet {
             blinded_public_shares,
             private_shares,
             blinder: scalar_to_biguint(&wallet.blinder),
+            update_locked: wallet.update_locked.load(Ordering::Relaxed),
```

Comment:
@joeykraut Let me know if I should do `update_locked: wallet.is_locked()` here. Purposefully didn't because of build errors saying `wallet.fees` partially moved due to this method call on line 89.

Reply:
Yeah probably best to use `wallet.is_locked`. Just do `let update_locked = wallet.is_locked();` above the line where the move happens